packages/popover/src/Popover.js

```diff
@@ -59,13 +59,14 @@ class Popover extends Component {
         <Reference>
           {({ ref }) => (
             // Need to be a native element, because of ref forwarding limitations with DS functional components
-            <InlineContainer innerRef={ref}>
+            // Styled-Components v4 uses regular refs, keep innerRef for v3 peer dependency
```

Comment:
Nit: Feel that the comment could be either ✂️, as google searching the topic styled-components v4 could land on some reference.

Or, the comment could become:
`// TODO fixme: when pcln-design-system future upgrade decides to drop support of innerRef prop as React legacy support is safe to deprecate, the innerRef prop here could be removed.`

Reply:
Removed the comments